Documentation/contributing.rst

```diff
@@ -1274,76 +1540,269 @@ simplify our Jenkinsfiles. The exported methods are:
+
+Backport criteria for X.Y.Z+n
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Criteria for the inclusion into latest stable release branch, i.e. what goes
+into ``v1.1.x`` before ``v1.2.0`` has been released:
+
+- All bugfixes
+
+Backport criteria for X.Y-1.Z
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Criteria for the inclusion into stable release branches are:
+Criteria for the inclusion into the stable release branch of the previous
+release, i.e. what goes into ``v1.0.x``, before ``v1.2.0`` has been released:
 
 - Security relevant fixes
 - Major bugfixes relevant to the correct operation of Cilium
 
+.. _lts:
 
-Steps to release
-~~~~~~~~~~~~~~~~
+LTS
+~~~
+
+The group of committers nominates a release to be a long term stable release.
+Such releases are guaranteed to receive backports for major and security
+relevant bugfixes. LTS releases will be declared end of life after 6 months.
+The group of committers will nominate and start supporting a new LTS release
+before the current LTS expires. If for some reason, no release can be declared
+LTS before the current LTS release expires, the current LTS lifetime will be
+extended.
+
+Given the current 6 weeks release cadence, the development teams will aim at
+declaring every 3rd release to be an LTS to guarantee enough time overlap
+between LTS release.
+
+Current LTS releases
+^^^^^^^^^^^^^^^^^^^^
+
++----------------------+---------------------------+-----------------------+
+| Release              | Original Release Date     | Scheduled End of Life |
++======================+===========================+=======================+
+| 1.0                  | 2018-04-24                | 2018-10-24            |
++----------------------+---------------------------+-----------------------+
+
+.. _generic_release_process:
+
+Generic Release Process
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This process applies to all releases other than minor releases, this includes:
+
+* Stable releases
+* Release candidates
+
+If you intent to release a new minor release, see the
+:ref:`minor_release_process` section instead.
+
+
+#. Ensure that the necessary backports have been completed and merged. See
+   :ref:`backport_process`.
+#. Checkout the desired stable branch and pull it:
+
+   ::
 
-1. Create a new automated build for ``cilium/cilium`` container image tag with
-   the full Cilium version on hub.docker.com. Point the automated build to the
-   development branch of the to be released version. This will ensure that the
-   to be released version always has a corresponding container image tag
-   assigned.
-2. Update the AUTHORS file by running ``make update-authors``
-3. Update the ``cilium_version`` and ``cilium_tag`` variables in
+       git checkout v1.0; git pull
+
+#. Create a branch for the release pull request:
+
+   ::
+   
+       git checkout -b pr/prepare-v1.0.3
+
+#. Update the ``VERSION`` file to represent ``X.Y.Z+1``
+#. Update the image tag versions in the examples:
+
+   ::
+   
+       make -C examples/kubernetes clean all
```

Comment:
Change the `PullPolicy` to `IfNotPresent` in the .sed files

Reply:
Good catch (again) :-)